js/packages/teams-ai/src/MessageExtensions.ts

```diff
@@ -24,34 +24,34 @@ import { Application, RouteSelector, Query } from './Application';
 import { TurnState } from './TurnState';
 
 /**
- * @private
+ * @public
  */
-const ANONYMOUS_QUERY_LINK_INVOKE_NAME = `composeExtension/anonymousQueryLink`;
+export const ANONYMOUS_QUERY_LINK_INVOKE_NAME = `composeExtension/anonymousQueryLink`;
```

Comment:
Why do these need to be exported?

Reply:
It's being used in `MessagingExtensionAuthentication` class to check if incoming invoke activity is one of these types.

Reply:
I copy-pasted these values in MessageExtensions.spec.ts. since there's three places now that they're being used, I'd prefer if we made an enum.

Reply:
fixed